src/ui/components/ShowMoreCard/index.js

```diff
@@ -16,30 +16,35 @@ export class ShowMoreCardBase extends React.Component {
     header: PropTypes.node,
     footer: PropTypes.node,
     i18n: PropTypes.object.isRequired,
+    maxHeight: PropTypes.number.isRequired,
+  }
+
+  static defaultProps = {
+    maxHeight: 100,
```

Comment:
I'm fine with passing this in but since we don't know if we'll always want this I'd say it's probably best to leave it out for now.

My reasoning is that at some point we might want to have "sizes" rather than "heights". So on mobile we might want this to be 100px but maybe on desktop we want 200px but we want that to be the "small" card.